### DISCUSSION TITLE
Hourly? Audio file creation behavior

### DISCUSSION THREAD

**Discussion by LoganSound** [rtl-airband/RTLSDR-Airband#211] [Q&A]
I am using inotifywait to rename and process audio files with the time they are created. It appears files are only created or perhaps finalized at the top of each hour (:00) is this configurable? I would love to have the file finalized a number of seconds after the squelch has closed.

**Comment by szpajder** [comment]
If hourly rotation is is use, it can currently only happen at the top of the hour, this is hardcoded.

Alternatively you may set `split_on_transmission = true` in the output configuration section to store every transmission in a separate file (this mode is coupled with squelch operation).

**Reply by LoganSound** [reply]
Thanks! And thanks for making such a fantastic tool!
